File: Prod.RutaDigital.Entidades/PerfilAvance/PremioConsumoResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Prod.RutaDigital.Entidades
{
    public class PremioConsumoResponse
    {
        public int? id_premio { get; set; }
        public string? nombre_premio { get; set; }
        public string? descripcion { get; set; }
        public string? foto { get; set; }
        public Int16? puntos_consumo { get; set; }
        public DateTime? fecha_consumo { get; set; }
    }
}

File: Prod.RutaDigital.Presentacion.Angular/ClientApp/src/app/perfil-avance/interfaces/premio-consumo.response.ts
import { SafeUrl } from '@angular/platform-browser';

export interface PremioConsumoResponse {
  id_premio?: number;
  nombre_premio?: string;
  descripcion?: string;
  foto?: string;
  puntos_consumo?: number;
  fecha_consumo?: Date;
  numArray?: Blob;
  imagenPremio?: SafeUrl;
}
